deny.toml

```diff
@@ -2,7 +2,7 @@
 # examples sometimes lag behind as they use higher level dependencies
 # we exclude these from the check because they are not part of our
 # actual dependency tree
-exclude = ["kube-examples"]
+exclude = ["kube-examples", "rustls-native-certs"]
```

Comment:
oh uh, that's .. i didn't know you could do it here. this is usually for our immediate workspace members

Reply:
i was thinking

```
[[bans.skip]]
name = "rustls-native-certs"
```

below

Reply:
oh, I've seen graph soooo lol

Reply:
it seem this solves the issue too:
```patch
 [bans]
-multiple-versions = "deny"
+multiple-versions = "allow"
```

Reply:
and this too, and it's probably what you wanted ^^;;
```patch
+[[bans.skip]]
+name = "rustls-native-certs"
```

Reply:
yeah, the multiple-version deny is __sometimes__ useful. it does lead to a bunch of skip listing like this though.